src/DBConnector/Connector.java

```diff
@@ -75,49 +113,56 @@ public void printAllUsers() {
    */
   public boolean addAccount(String username, String password) {
 
-    /* string constructed for database */
-    String sqlAdd = "insert into "+tableName + "(username, password)" + "values(" + "'" + username
-        + "'" + "," + "'" + password + "'" + ")";
```

Comment:
This would read more easily if you used a `Joiner.on("'").join(fields)`
https://google.github.io/guava/releases/16.0/api/docs/com/google/common/base/Joiner.html